egs2/chime8_task1/asr1/run.sh

```diff
@@ -0,0 +1,305 @@
+    --nbpe $nbpe \
+    --bpe_nlsyms "${bpe_nlsyms}" \
+    --nlsyms_txt "data/nlsyms.txt" \
+    --feats_type raw \
+    --feats_normalize utterance_mvn \
+    --audio_format "flac" \
+    --min_wav_duration $train_min_segment_length \
+    --max_wav_duration $train_max_segment_length \
+    --speed_perturb_factors "0.95 1.0 1.1" \
+    --asr_config "${asr_config}" \
+    --inference_config "${inference_config}" \
+    --use_lm ${use_lm} \
+    --lm_config "${lm_config}" \
+    --inference_asr_model ${inference_asr_model} \
+    --use_word_lm ${use_word_lm} \
+    --word_vocab_size ${word_vocab_size} \
+    --train_set "${asr_train_set}" \
+    --valid_set "${asr_cv_set}" \
+    --test_sets "${asr_tt_set}" \
+    --bpe_train_text "data/${asr_train_set}/text" \
+    --lm_train_text "data/${asr_train_set}/text" ${pretrained_affix}
+fi
+
+if [ "${run_on}" == "eval" ]; then
+  log "Scoring not available for eval set till the end of the challenge."
+  exit
+fi
+
+if [ ${stage} -le 4 ] && [ ${stop_stage} -ge 4 ]; then
+  # final scoring
+  log "Scoring ASR predictions for CHiME-7 DASR challenge."
+  # note, we re-create the asr exp folder here based on asr.sh
+  if [ -n "$use_pretrained" ]; then
+    asr_exp="exp/${use_pretrained}"
+  else
+    #FIXME ask kamo ? if this fails put your own !
+    asr_tag=${asr_tag} #"$(basename "${asr_config}" .yaml)_raw"
+    asr_exp="exp/asr_${asr_tag}"
+  fi
+  inference_tag="$(basename "${inference_config}" .yaml)"
+  inference_tag+="_asr_model_$(echo "${inference_asr_model}" | sed -e "s/\//_/g" -e "s/\.[^.]*$//g")"
+
+  # when a dataset is both in cv and tt in ESPNet2, the tt dataset is
+  # placed unmodified into org. we create a symbolic link
+  # so it can be parsed as the other datasets.
+  for tt_dset in $asr_tt_set; do
+      if [ ! -e "${asr_exp}/${inference_tag}/${tt_dset}" ] && [ -d  "${asr_exp}/${inference_tag}/org/${tt_dset}" ]; then
+        # Creating the parent directory
+        mkdir -p "${asr_exp}/${inference_tag}/${tt_dset}" && rmdir "${asr_exp}/${inference_tag}/${tt_dset}"
+        ln -sf "$(cd ${asr_exp}/${inference_tag}/org/${tt_dset}; pwd)" "${asr_exp}/${inference_tag}/${tt_dset}"
+      fi
+  done
+
+  for tt_dset in $asr_tt_set; do
+    split="$(cut -d'/' -f3 <<<${tt_dset})"
+    dset_name="$(cut -d'/' -f2 <<<${tt_dset})"
+    if [ ${dset_name} == mixer6 ]; then
+      regex="([0-9]+_[0-9]+_(LDC|HRM)_[0-9]+)" # different session naming
+    else
+      regex="(S[0-9]+)"
+    fi
+    python local/asr2json.py -i ${asr_exp}/${inference_tag}/${tt_dset}/text -o ${asr_exp}/${inference_tag}/chime7dasr_hyp/$split/$dset_name -r $regex
+    # the content of this output folder is what you should send for evaluation to the
+    # organizers.
+  done
+  #FIXME add chime-utils scoring here !
+  split=dev # participants cannot evaluate eval
+  LOG_OUT=${asr_exp}/${inference_tag}/scoring/scoring.log
+  python local/da_wer_scoring.py -s ${asr_exp}/${inference_tag}/chime7dasr_hyp/$split \
```

Comment:
Will you keep using DA WER?

Reply:
I fixed scoring. 
I am printing tcpWER and cpWER now via chime-utils package.